snomed/com.b2international.snowowl.snomed.datastore/src/com/b2international/snowowl/snomed/datastore/converter/BaseSnomedComponentConverter.java

```diff
@@ -97,11 +95,11 @@ protected final Date toEffectiveTime(final long effectiveTimeAsLong) {
 		return EffectiveTimes.toDate(effectiveTimeAsLong);
 	}
 	
-	protected final int getLimit(final Options expandOptions) {
+	protected static final int getLimit(final Options expandOptions) {
 		return expandOptions.containsKey("limit") ? expandOptions.get("limit", Integer.class) : 50;
```

Comment:
Yeah, it should, but I don't see any. 😄 I'll create the constant: `com.b2international.snowowl.datastore.request.SearchRequestBuilder.DEFAULT_LIMIT`.